yal/assessments.py

```diff
@@ -441,6 +441,15 @@ async def state_handle_assessment_reminder_response(self):
             return await self.go_to_state("state_pre_mainmenu")
 
     async def state_not_interested(self):
```

Comment:
Is this state only used in the endline survey?

Reply:
It is also used on the baseline

Reply:
Oh ok, I guess its fine to update "endline_survey_started" to "" at that point too